tests/fixtures/services.py

```diff
@@ -201,7 +201,7 @@ def wired(self) -> Generator[None, None, None]:
         self.services.unwire()
 
 
-@pytest.fixture(autouse=True)
+@pytest.fixture
```

Comment:
This fixture was set to `autouse` before, but that isn't necessary. It just needs to be added to the alembic fixture. I'd like to avoid `autouse` fixtures if we can, so you always know the fixtures that are active when calling a function.